__tests__/app.test.js

```diff
@@ -157,3 +163,52 @@ describe("GET/api/articles/:article_id/comments", () => {
     });
   });
 });
+describe("POST/api/articles/:article_id/comments", () => {
+  test("should return the comment from the user with the correct properties and a status 201", () => {
+    const inputComment = {
+      username: "lurker",
+      body: "this is a new comment",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(inputComment)
+      .expect(201)
+      .then((response) => {
+       
+        const newComment = response.body.comment;
+        expect(newComment).toMatchObject({
+          body: expect.any(String),
+          votes: expect.any(Number),
+          author: expect.any(String),
+          article_id: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
```

Comment:
We could add a test where we check what happens if we add extra properties to the inputComment. Ideally it should still post it with a 201